core/third_party_libs/dompdf/lib/fonts/dompdf_font_family_cache.dist.php

```diff
@@ -1,72 +1,67 @@
-    'italic' => DOMPDF_FONT_DIR . 'Courier-Oblique',
-    'bold_italic' => DOMPDF_FONT_DIR . 'Courier-BoldOblique',
-  ),
-  'fixed' =>
-  array (
-    'normal' => DOMPDF_FONT_DIR . 'Courier',
-    'bold' => DOMPDF_FONT_DIR . 'Courier-Bold',
-    'italic' => DOMPDF_FONT_DIR . 'Courier-Oblique',
-    'bold_italic' => DOMPDF_FONT_DIR . 'Courier-BoldOblique',
-  )
-) ?>
+<?php
+$distFontDir = $rootDir . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'fonts' . DIRECTORY_SEPARATOR;
+return array(
+    'sans-serif' =>
+        array(
+            'normal' => $distFontDir . 'Helvetica',
+            'bold' => $distFontDir . 'Helvetica-Bold',
+            'italic' => $distFontDir . 'Helvetica-Oblique',
+            'bold_italic' => $distFontDir . 'Helvetica-BoldOblique'
+        ),
+    'times' =>
+        array(
+            'normal' => $distFontDir . 'Times-Roman',
+            'bold' => $distFontDir . 'Times-Bold',
+            'italic' => $distFontDir . 'Times-Italic',
+            'bold_italic' => $distFontDir . 'Times-BoldItalic'
+        ),
+    'times-roman' =>
+        array(
+            'normal' => $distFontDir . 'Times-Roman',
+            'bold' => $distFontDir . 'Times-Bold',
+            'italic' => $distFontDir . 'Times-Italic',
+            'bold_italic' => $distFontDir . 'Times-BoldItalic'
+        ),
+    'courier' =>
+        array(
+            'normal' => $distFontDir . 'Courier',
+            'bold' => $distFontDir . 'Courier-Bold',
+            'italic' => $distFontDir . 'Courier-Oblique',
+            'bold_italic' => $distFontDir . 'Courier-BoldOblique'
+        ),
+    'helvetica' =>
+        array(
+            'normal' => $distFontDir . 'Helvetica',
+            'bold' => $distFontDir . 'Helvetica-Bold',
+            'italic' => $distFontDir . 'Helvetica-Oblique',
+            'bold_italic' => $distFontDir . 'Helvetica-BoldOblique'
+        ),
+    'symbol' =>
+        array(
+            'normal' => $distFontDir . 'Symbol',
+            'bold' => $distFontDir . 'Symbol',
+            'italic' => $distFontDir . 'Symbol',
+            'bold_italic' => $distFontDir . 'Symbol'
+        ),
+    'serif' =>
+        array(
+            'normal' => $distFontDir . 'Times-Roman',
+            'bold' => $distFontDir . 'Times-Bold',
+            'italic' => $distFontDir . 'Times-Italic',
+            'bold_italic' => $distFontDir . 'Times-BoldItalic'
+        ),
+    'monospace' =>
+        array(
+            'normal' => $distFontDir . 'Courier',
+            'bold' => $distFontDir . 'Courier-Bold',
+            'italic' => $distFontDir . 'Courier-Oblique',
+            'bold_italic' => $distFontDir . 'Courier-BoldOblique'
+        ),
+    'fixed' =>
+        array(
+            'normal' => $distFontDir . 'Courier',
+            'bold' => $distFontDir . 'Courier-Bold',
+            'italic' => $distFontDir . 'Courier-Oblique',
+            'bold_italic' => $distFontDir . 'Courier-BoldOblique'
+        )
+);
```

Comment:
we used to have zpfdingbats in here. We should probably re-add it, as I think it helped with emojis or something

Reply:
ah ok, ya that's right, we removed zapdingbats font files... we just erroneously didn't remove it from the font cache file. So you're right to remove it from both places then :+1:

Reply:
So we don't want it after all huh ? Revert the `add zapfdinbats back to the core fonts list` back ?

Reply:
oh yeah, well we don't have the zapfdinbats font files anyway. So yes I suppose that commit should be reverted